apps/api/src/bootstrap.ts

```diff
@@ -39,6 +40,8 @@ if (process.env.SENTRY_DSN) {
 validateEnv();
 
 export async function bootstrap(expressApp?): Promise<INestApplication> {
+  startupDistributedLock();
```

Comment:
We add the initalisation of the distributed lock.
We should add the shutdown of the distributed lock (implemented but not integrated). @scopsy  @Cliftonz where do we do the shutdown of the Nest server? I can't find it.

Reply:
Could we maybe move this logic to shared.module.ts: 
```
{
    provide: DalService,
    useFactory: async () => {
      await dalService.connect(process.env.MONGO_URL);

      return dalService;
    },
  },
```

And follow a similar pattern of the Dal service so than nest will bootstrap it as part of it's lifecycle of modules instead of here?